packages/docusaurus-theme-classic/src/theme/BlogPostItem/Header/Info/index.tsx

```diff
@@ -79,6 +81,13 @@ export default function BlogPostItemHeaderInfo({
         <>
           <Spacer />
           <ReadingTime readingTime={readingTime} />
+          <Spacer />
+          {(lastUpdatedAt || lastUpdatedBy) && (
+            <LastUpdated
+              lastUpdatedAt={lastUpdatedAt}
+              lastUpdatedBy={lastUpdatedBy}
+            />
+          )}
```

Comment:
We agreed on the issue that we want to make it uniform with docs and add this to the footer instead of header 

![image](https://github.com/facebook/docusaurus/assets/749374/d6187c06-3fde-4659-a0b1-faaa8648c8bd)


It won't please everyone but will at least give a consistent UX, and give the opportunity for unsatisfied users to swizzle and fix the design if needed